nuget/NUnit3TestAdapter.nuspec

```diff
@@ -24,6 +24,19 @@
     <tags>test visualstudio testadapter nunit nunit3 dotnet</tags>
 
     <developmentDependency>false</developmentDependency>
+
+    <dependencies>
+      <group targetFramework="net462">
+        <dependency id="Microsoft.Testing.Extensions.VSTestBridge" version="1.0.1" />
+        <dependency id="Microsoft.Testing.Extensions.Telemetry" version="1.0.1" />
```

Comment:
This is not a requirement but we wanted to avoid having telemetry on the core platform as it could be causing issues and would be leading to extra deps on the platform that we want to keep as light and no deps as possible.

Reply:
What does this telemetry package do?
If it sends information to some central server, we're not very keen of having that into the adapter.  In understand it is optional, but if any telemetry should be sent, it should be to the NUnit project, and we don't have any server for receiving telemetry.

Reply:
This is collecting "general" data about the platform usage (see https://learn.microsoft.com/dotnet/core/testing/unit-testing-mstest-runner-telemetry#data-points). This is something that is backed in VSTest, SDK... most MS tools but in our case we wanted to ensure that the core platform is not having any external deps so we had to move the telemetry into its own extension point.

We are suggesting to keep it, although I understand fear/concerns..., mainly because this is helping us justify priorization of work to management. Especially now that we have 2 competing platforms.